backend/app/api/endpoints/projects.py

```
@@ -88,6 +90,32 @@ def create_project_endpoint(
         )
 
 
+@router.post(
+    "/git-workspace",
+    response_model=GitWorkspaceProjectResponse,
+    status_code=status.HTTP_201_CREATED,
+)
+async def create_git_workspace_project_endpoint(
+    project_create: GitWorkspaceProjectCreate,
+    current_user: User = Depends(get_current_user),
```

Ruff (0.15.15)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)